SPECS/bpftrace/bpftrace.spec

```diff
@@ -14,6 +14,7 @@ BuildRequires:  bison
 BuildRequires:  cereal-devel
 BuildRequires:  clang-devel
 BuildRequires:  cmake
+BuildRequires:  dwarves
```

Comment:
I guess this could go below under `with_check` ?

Reply:
Ack, yes will gate it :)